### ISSUE TITLE
Add support for dependency caching in AWS Codebuild

### ISSUE THREAD

**Issue by vladgolubev** [terraform-providers/terraform-provider-aws#2401]
### References

* Announcement - https://aws.amazon.com/about-aws/whats-new/2017/11/aws-codebuild-adds-new-capabilities-to-support-amazon-virtual-private-cloud-resources-and-dependency-caching/
* Release blog post - https://aws.amazon.com/blogs/devops/how-to-enable-caching-for-aws-codebuild/

**Comment by bflad** [comment]
Prerequisite: #2403

**Comment by squidfunk** [comment]
Any updates on this?

**Comment by ndench** [comment]
Duplicate #2842 ?

**Comment by bflad** [comment]
This support has been merged into master via #2860 and will release with v1.14.0 of the AWS provider, likely later today. 🎉
